src/main/kotlin/org/rust/openapiext/CommandLineExt.kt

```diff
@@ -55,7 +55,10 @@ fun GeneralCommandLine.execute(
     val cargoKiller = Disposable {
         // Don't attempt a graceful termination, Cargo can be SIGKILLed safely.
         // https://github.com/rust-lang/cargo/issues/3566
-        handler.destroyProcess()
+        if (!handler.isProcessTerminated) {
+            handler.process.destroyForcibly() // Send SIGKILL
+            handler.destroyProcess()
```

Comment:
Btw I'm not sure we need this line at all. Can it be useful if `listener` is present?